### PR TITLE
Add support for Sinope smart dimmer DM2500ZB

### PR DESCRIPTION
**Pull request by bvanheu**
Please bare with me, I've cargo-cult the `configure` until something worked with my device :)

I'm not sure if genBasic, genIdentify nor genGroups are needed.

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -9119,6 +9119,23 @@ const devices = [
...
+        zigbeeModel: ['DM2500ZB'],
+        model: 'DM2500ZB',
+        vendor: 'Sinope',
+        description: 'Zigbee smart dimmer',
+        extend: generic.light_onoff_brightness,
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            const binds = [
+                'genBasic', 'genIdentify', 'genGroups', 'genOnOff', 'genLevelCtrl',
```
Comment (Koenkk) [inline devices.js:9132]: `genBasic`, `genIdentify`, `genGroups` can be removed here.

Comment (bvanheu) [reply devices.js]: Done.

Comment (bvanheu) [reply devices.js]: Should I squash both commit + force push? I'm not familiar with Github pull-request workflow.

Comment (Koenkk) [reply devices.js]: Not needed, I can squash it from my side, thanks!

**Pull request merged**